src/main/common/huffman.h

```diff
@@ -24,8 +24,8 @@
 
 #define HUFFMAN_TABLE_SIZE 257 // 256 characters plus EOF
 typedef struct huffmanTable_s {
-    uint8_t     codeLen;
-    uint16_t    code;
+    uint16_t    code:12;
```

Comment:
Maybe swapping `code` and `codeLen` (on little endian machine) will save instruction or two in decoding (using just bitmask instead of shift)

Reply:
4B saved. I will change it.